compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -379,6 +379,7 @@ public class Desugar extends BLangNodeVisitor {
     private TransactionDesugar transactionDesugar;
     private ObservabilityDesugar observabilityDesugar;
     private Code2CloudDesugar code2CloudDesugar;
+    private RuntimeManagementDesugar runtimeManagementDesugar;
```

Comment:
Shall we rename all occurence of runtimeManagemnt to remoteManagement

Reply:
removed